Formula/guile.rb

```diff
@@ -35,26 +41,38 @@ class Guile < Formula
   depends_on "pkg-config" # guile-config is a wrapper around pkg-config.
   depends_on "readline"
 
+  ### Remove after configure.ac patch is not necessary anymore.
+  uses_from_macos "flex" => :build
+  ###
+
   uses_from_macos "gperf"
   uses_from_macos "libffi", since: :catalina
   uses_from_macos "libxcrypt"
 
+  on_macos do
+    # A patch to fix JIT on Apple Silicon is embedded below, this fixes:
+    #   https://debbugs.gnu.org/cgi/bugreport.cgi?bug=44505
+    # The patch has been accepted upstream.
+    #   https://git.savannah.gnu.org/cgit/guile.git/commit/?id=3bdcc3668fd8f9a5b6c9a313ff8d70acb32b2a52
```

Comment:
Why not use the upstream patch then?

Reply:
Oooops! Let me check how to do that. Thanks!